package.json

```diff
@@ -1,18 +1,16 @@
 {
   "name": "@digicatapult/sqnc-identity-service",
-  "version": "1.11.16",
+  "version": "1.12.0",
   "description": "Identity Service for SQNC",
   "type": "module",
   "main": "app/index.js",
   "scripts": {
     "test": "NODE_ENV=test mocha --config ./test/mocharc.json -r esm ./test",
-    "test:jwt": "NODE_ENV=test AUTH_TYPE=JWT mocha --config ./test/mocharc.json -r esm ./test",
     "lint": "eslint .",
     "depcheck": "depcheck",
     "start": "NODE_ENV=production node app/index.js",
     "dev": "NODE_ENV=development nodemon app/index.js | pino-colada",
-    "coverage": "LOG_LEVEL=fatal NODE_ENV=development c8 mocha --recursive ./test/integration -r esm --timeout 60000 --slow 20000 --exit",
-    "coverage:merge": "LOG_LEVEL=fatal NODE_ENV=development c8 --no-clean npm run test && c8 --no-clean npm run test:jwt && c8 merge .c8_output --timeout 60000 --slow 20000 --exit"
```

Comment:
Why did you get rid of this?

Reply:
Previously we had to do test runs to get the full coverage statistics. One for "jwt" auth one for "none". Now we only have the one type we only need to do one test run and we don't need to do the merge